docs/monitoring.md

```diff
@@ -1006,6 +1009,37 @@ when running in local mode.
 
 - namespace=JVMCPU
   - jvmCpuTime
+  
+- namespace=MemoryMetrics
+  - **note:** MemoryMetrics counters are updated as part the executor metrics heart 
```

Comment:
Nit: In the `executor metrics heart beat` I do not think the `metrics` word is correct there it is `executor's heartbeat` (I have only seen heartbeat written as one word i.e. at the config description of [spark.executor.heartbeatInterval](https://spark.apache.org/docs/latest/configuration.html)).

Reply:
Good points thanks.